pytorch_lightning/utilities/cloud_io.py

```diff
@@ -33,7 +36,7 @@ def load(path_or_url: Union[str, IO, Path], map_location=None):
         return torch.load(f, map_location=map_location)
 
 
-def get_filesystem(path: Union[str, Path]):
+def get_filesystem(path: Union[str, Path]) -> LocalFileSystem:
```

Comment:
This can be something other than `LocalFileSystem`, a better return type would be `AbstractFileSystem`: https://github.com/intake/filesystem_spec/blob/bdcdb304bea87c2d458c6045cfbf28d963fa715b/fsspec/spec.py#L97